internal/kokoro/check_incompat_changes.sh

```diff
@@ -6,13 +6,17 @@ set -x
 # Only run apidiff checks on go1.11 (we only need it once).
```

Comment:
Should probably change this comment if we are changing the check to 1.12 instead of 1.11

Unless the logic should change ?